client/post-editor/editor-revisions-list/item.jsx

```diff
@@ -88,7 +88,7 @@ EditorRevisionsListItem.propTypes = {
 
 	// connected to state
 	isMultiUserSite: PropTypes.bool.isRequired,
-	revisionChanges: PropTypes.array.isRequired,
+	revisionChanges: PropTypes.object.isRequired,
```

Comment:
I checked the file for `revisionChanges` usage and all accessors of it's properties are safe (`get()`) so `object` works here without shape :)

Thanks for the fix!